home.css

```diff
@@ -166,6 +186,22 @@ button {
   z-index: 50;
   background-color: #cc8c85;
```

Comment:
According to the project requirements, the mobile menu's background color should be different than the one you currently have.

<img width="571" alt="Screen Shot 2022-07-27 at 9 06 18" src="https://user-images.githubusercontent.com/82426682/181267794-6e20c39a-e0aa-4981-b6ae-bfd5238c04a4.png">

Please fix the background color for both pages.